src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs

```diff
@@ -1097,6 +1097,9 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Autom
                                                 $offsetInLine = $myinv.OffsetInLine - 1
                                             }
 
+                                            if (-not $line.EndsWith(""`n"")) {
+                                                $line += ""`n""
```

Comment:
Should not it be environment dependent new line? E.g. `System.Environment.NewLine`?
Output is redirected to files sometimes and then opened by Notepad which still does not like `\n`.

Reply:
Good point.  Will update.